clouddriver-aws/src/integration/java/com/netflix/spinnaker/clouddriver/aws/test/CreateServerGroupSpec.java

```diff
@@ -87,7 +89,9 @@ public class CreateServerGroupSpec extends AwsBaseSpec {
   private AmazonEC2 mockEc2 = mock(AmazonEC2.class);
 
   @BeforeEach
-  public void setup() {
+  private void init(TestInfo testInfo) {
+    System.out.println("--------------- Test " + testInfo.getDisplayName());
```

Comment:
System.out?  Would... ideally be a logger log I'd think...

Reply:
Perhaps, but this way we still see the messages in the face of logging troubles.